corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js

```diff
@@ -833,7 +855,21 @@ cloudCare.AppView = Backbone.View.extend({
         $('#webforms').html("");
         this.caseSelectionView.model.set("parentCase", null);
         this.caseSelectionView.model.set("childCase", null);
-    }
+    },
+    syncDb: function (username) {
+        var self = this;
```

Comment:
normally people do this so they can access the original `this` in closures, but in this case you don't reference it so i think it's fine to remove